docusaurus.config.en.js

```diff
@@ -46,14 +46,8 @@ const config = {
   ],
   // Settings for Docusaurus Faster - build optimizations
   future: {
-    experimental_faster: {
-      swcJsLoader: true,
-      swcJsMinimizer: true,
-      swcHtmlMinimizer: true,
-      lightningCssMinimizer: true,
-      rspackBundler: true,
-      mdxCrossCompilerCache: true,
-    },
+    experimental_faster: true,
```

Comment:
with all flags on we can replace it with a single command. `v4: true` for Docusaurus v4 future flags. See https://docusaurus.io/blog/releases/3.8#future-flags